aether-kernel/entrypoint.sh

```diff
@@ -138,7 +138,8 @@ case "$1" in
     ;;
 
     start )
-        setup
+        # check if required environment variables were set
+        ./conf/check_vars.sh
```

Comment:
Thinking on this issue a little bit more... could you use a condition here in order to keep local development working?

```bash
if [ "IS THIS A PROD DEPLOYMENT" ]; then
    ./conf/check_vars.sh
else
    setup
fi
```

Otherwise this is going to stop working as expected :scream:

Reply:
Correct me if I'm wrong but the `start` command, afaik, is not used for local development, and if you did it would only fail on a fresh install of aether.  What do you think if instead I add a mention to the `README` file stating explicitly that `start_dev` should be used for local development (which is the current state of things anyway)?

I would prefer that over adding a new flag or environment variable that we will have to keep track of. What do you think?

Reply:
The migration command should be run every time there is a change in the data model not only with a clean start. But yes, this is something that doesn't, or shouldn't, happen quite often.

`aether-bootstrap` and `gather-deploy` use the `start` command, if you change this we should spread this change to them and, also change the documentation (gh-pages).

@adamvert what do you think?